### DISCUSSION TITLE
Univariable vs. multivariable regression in Cox’s proportional hazard model

### DISCUSSION THREAD

**Discussion by giemmecci** [CamDavidsonPilon/lifelines#1417] [Q&A] [answered]
Hi,
Thank you for developing and sharing this package.

I have a basic question (I'm new to survival analysis).

If we run the [Cox's proportional hazard model](https://lifelines.readthedocs.io/en/latest/Survival%20Regression.html#cox-s-proportional-hazard-model) in lifelines, and we don't specify a formula, are the results referring to multiple univariable regression models run on each of the variables, or a multivariable regression model?

In other words, how can we run univariable and multivariable regressions in lifelines?

My understanding is that formulas allow you to select the variable to use (among other things), so in that case, we should interpret the results as a multivariable regression model. Is this correct?

Thanks!

**Comment by CamDavidsonPilon** [comment, marked as answer]
Hi @giemmecci, good question

The `formula` is there to specify more precise transformation you may wish to do (think interaction terms, or quadratic terms). If no formula is specified, _all_ variables in the dataframe are used in a single regression (so multivariable regression model).

**Reply by giemmecci** [reply]
Thanks so much for the quick reply!

So to wrap up, if I run this:

`cph.fit(rossi, duration_col='week', event_col='arrest', formula="fin")`

Will the result correspond to a univariable regression model run on the "fin" variable?

Thanks!

**Comment by CamDavidsonPilon** [comment]
Yup, that's exactly right.

**Reply by giemmecci** [reply]
Thank you!
